niworkflows/interfaces/segmentation.py

```diff
@@ -106,37 +106,64 @@ class MELODICOutputSpecRPT(reporting.ReportCapableOutputSpec,
     pass
 
 
-class MELODICRPT(reporting.ReportCapableInterface, fsl.MELODIC):
+class MELODICRPT(fsl.MELODIC):
     input_spec = MELODICInputSpecRPT
     output_spec = MELODICOutputSpecRPT
+    _out_report = None
+    generate_report = False
```

Comment:
Since this always set in `__init__`, no need to place it in the class attributes.

```suggestion
```